ReleaseNotes.md

```diff
@@ -11,6 +11,18 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 08/05/2020 -
+##### Version microsoft/oms:ciprod07152020 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod08052020 (linux)
+##### Version microsoft/oms:win-ciprod05262020-2 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod08052020 (windows)
+##### Code change log
```

Comment:
few more -
* Bug fix for showing node as 'not ready' when there is disk pressure
* oneagent integration (disabled by default)
* Add region check before sending alertable metrics to MDM
* Telemetry fix for agent telemetry for sov. clouds

Reply:
> * Bug fix for showing node as 'not ready' when there is disk pressure
> * oneagent integration (disabled by default)
> * Add region check before sending alertable metrics to MDM
> * Telemetry fix for agent telemetry for sov. clouds

> Fixed.